tests/lib/c_lib/common/src/test_sqrt.c

```diff
@@ -147,6 +149,9 @@ int32_t *p_root_squared = (int32_t *)&root_squared;
 
 ZTEST(libc_common, test_sqrt)
 {
+#if __SIZEOF_DOUBLE__ != 8
```

Comment:
Maybe a bit of overthinking:
gcc and clang define this, but I wonder about other compilers. Maybe skipping the test is not that bad for such a possible compile anyhow.

Reply:
and just checking sizeof(double) is simpler anyways. Thanks!.